.github/workflows/ci.yml

```diff
@@ -43,8 +43,8 @@ jobs:
           bundler-cache: true
           cache-version: 7
 
-      - name: Check if documentation is up to date
-        run: bundle exec rake ruby_lsp:check_docs
+      # - name: Check if documentation is up to date
+      #   run: bundle exec rake ruby_lsp:check_docs
```

Comment:
TODO: look into this

Reply:
t was because the docs task requires the `server`, which hangs the task. I've pushed a fix.